### ISSUE TITLE
Failed to install 0.13.5

### ISSUE THREAD

**Issue by seancdavis** [eduardoboucas/esbuild#4]
I'm trying to install netlify-cli using pnpm, but am running into an issue with this package. It seems to be failing on the postinstall script.

Here is the trace:

```
.../node_modules/@netlify/esbuild postinstall$ node install.js
│ Trying to install "@netlify/esbuild-darwin-64" using npm
│ Failed to install "@netlify/esbuild-darwin-64" using npm: Command failed: npm install --loglevel=error --prefer-offline --no-audit --progress=false @netlify/esbuild-darwin-64@0.13.5
│ npm ERR! code ETARGET
│ npm ERR! notarget No matching version found for @netlify/esbuild-darwin-64@0.13.5.
│ npm ERR! notarget In most cases you or one of your dependencies are requesting
│ npm ERR! notarget a package version that doesn't exist.
│ npm ERR! A complete log of this run can be found in:
│ npm ERR!     [...]/.npm/_logs/2021-06-12T11_47_30_428Z-debug.log
│ Trying to download "https://registry.npmjs.org/@netlify/esbuild-darwin-64/-/@netlify/esbuild-darwin-64-0.13.5.tgz"
│ Failed to download "https://registry.npmjs.org/@netlify/esbuild-darwin-64/-/@netlify/esbuild-darwin-64-0.13.5.tgz": Server responded with 404
│ Install unsuccessful
```

It appears to be trying to download a file here: https://registry.npmjs.org/@netlify/esbuild-darwin-64/-/@netlify/esbuild-darwin-64-0.13.5.tgz

I ran `npm view @netlify/esbuild-darwin-64`, and it appears the tarball is located here instead: https://registry.npmjs.org/@netlify/esbuild-darwin-64/-/esbuild-darwin-64-0.13.5.tgz

I am still able to install netlify-cli globally via NPM, but I'd like to be able to package it up because it's helping me get some other dependencies installed during the Netlify build.

**Comment by eduardoboucas** [comment]
Hi @seancdavis! Apologies for the slow response here.

This should be fixed with `@netlify/esbuild@0.13.6` and `netlify-cli@3.37.23`.

If you're still experiencing this problem, please let me know and I'll reopen the issue.

Thanks!

**Comment by seancdavis** [comment]
That did the trick! Thank you.
